electron-app/src/views/Dashboard.vue

```diff
@@ -1,46 +1,75 @@
 <template>
   <v-container>
-    <v-row>
-      <v-col>
-        <h1 class="page-header">Dashboard</h1>
+    <v-row style="margin: 0;">
+      <v-col cols="12">
+        <overall-balances></overall-balances>
       </v-col>
     </v-row>
-
-    <div class="dashboard__information-boxes">
-      <exchange-box
-        v-for="exchange in exchanges"
-        :key="exchange.name"
-        :name="exchange.name"
-        :amount="exchange.total"
-      ></exchange-box>
-      <information-box
-        v-if="!blockchainTotal.isZero()"
-        id="blockchain_box"
-        icon="fa-hdd-o"
-        :amount="blockchainTotal"
-        blockchain
-      ></information-box>
-      <information-box
-        v-if="!fiatTotal.isZero()"
-        id="banks_box"
-        icon="fa-university"
-        :amount="fiatTotal"
-      ></information-box>
-    </div>
-
-    <v-row>
-      <v-col cols="12">
-        <h1 class="page-header">All Balances</h1>
+    <v-row class="flex" style="margin: 0; justify-content: space-evenly;">
+      <v-col cols="12" md="4" lg="4">
+        <summary-card
+          name="exchange"
+          can-refresh
+          :is-loading="exchangeIsLoading"
+          @refresh="refreshBalance($event)"
+        >
+          <exchange-box-new
```

Comment:
do we still use the old exchange box? if not it doesn't make any sense that the new component is postfix with new

Reply:
No, just didn't want to delete anything before clearing it with you :)